index.html

```diff
@@ -4,41 +4,70 @@
+                <form class="form-sign-up form" name="human" id="form" method="post">
+                    <div>
+                        <h2 class="form-sign-up__title">Sign Up</h2>
+                    </div>
+
                     <div>
-                        <label class="form">First Name</label>
-                        <input type="text" class="form" name="first-name" />
-                        <span class="message form" data-for="first-name"></span>
+                        <label class="form form-sign-up__label">Username:</label>
+                        <input type="text" class="form" name="user-name" />
+                        <span class="message form" data-for="user-name"></span>
                     </div>
 
                     <div>
-                        <label class="form">Last Name</label>
-                        <input type="text" class="form" name="last-name" />
-                        <span class="message form" data-for="last-name"></span>
+                        <label class="form form-sign-up__label">Email:</label>
+                        <input type="email" class="form" name="user-email" />
+                        <span class="message form" data-for="user-email"></span>
                     </div>
 
                     <div>
-                        <label class="form">Age</label>
-                        <input type="number" class="form" name="age" />
-                        <span class="message form" data-for="age"></span>
+                        <label class="form form-sign-up__label">Password:</label>
+                        <div class="input__wrapper">
+                            <input
+                                type="password"
+                                class="form input__password js-password-input"
+                                name="user-password"
+                                id="js-first-password"
+                            />
+                            <button class="btn__pass js-btn-password" data-target="#js-first-password">
+                                <img src="/image/eye.png" alt="show password" />
+                            </button>
+                        </div>
+
+                        <span class="message form" data-for="user-password"></span>
+                    </div>
+
+                    <div>
+                        <label class="form form-sign-up__label">Confirm Password:</label>
+                        <div class="input__wrapper">
+                            <input
+                                type="password"
+                                class="form input__password js-password-input"
+                                name="user-password-confirm"
+                                id="js-second-password"
+                            />
+                            <button class="btn__pass js-btn-password" data-target="#js-second-password">
+                                <img src="/image/eye.png" alt="show password" />
```

Comment:
for better practice - use 2 different icons for password show/hide 
<img width="111" alt="image" src="https://github.com/Own3Dqq/form-validation/assets/38155972/e1578b8b-3d39-4bab-b228-b955c483253e">
<img width="118" alt="image" src="https://github.com/Own3Dqq/form-validation/assets/38155972/83bc90bd-272e-481d-a4a0-edba33675df5">